src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -436,14 +436,20 @@ units::length::inch_t DragonLimelight::CalcXTargetToRobot(units::angle::degree_t
     units::length::inch_t XDistance = units::length::inch_t((units::math::tan(units::angle::degree_t(90) + camPitch + tY) * mountHeight) + units::math::abs(camXOffset));
     if (GetCameraYaw() > units::degree_t(std::abs(90.0)))
     {
-        return -1.0 * XDistance;
+        return -1.0 * (XDistance + m_driveThroughOffset);
```

Comment:
does m_driveThroughOffset ever get updated (it looks like it is always 0.0)?

Reply:
The reason it is 0 is because it works, but if we want to drive through the note or target more we have the ability, we can get rid of it if you would like. But I don't think it is hurting us at this point